cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -265,8 +265,13 @@ function(px4_nuttx_add_export)
 		COMMAND ${MKDIR} -p ${nuttx_src}
 		COMMAND ${CP} -a ${CMAKE_SOURCE_DIR}/NuttX/. ${nuttx_src}/
 		COMMAND ${RM} -rf ${nuttx_src}/.git
+						  ${nuttx_src}/apps/.git
```

Comment:
@dagar 
I wonder if we could use a portable copy that can exclude the .git and save time and disk...